pkg/arvo/sys/arvo.hoon

```diff
@@ -25,7 +25,7 @@
 +$  arch  (axil @uvI)
 ++  axal
   |$  [item]
-  [fil=(unit item) dir=(map @ta $)] ::
+  [fil=(unit item) dir=(map @ta $)]
```

Comment:
Since you've already changed this line ... I think I was too cute with this, and the recursion should just be `(axal item)` (referencing `$` there depends on `|$`not having an hygienic expansion).

Reply:
That actually blows up the compiler, overflowing the stack.  I assume it's because list/lest/pole/tree have an explicit non-recursive case (`~`), and this doesn't.

Reply:
My bad, I only tested it in the dojo. It's obviously not pressing -- spec hygiene will be its own effort.